werkit/aws_lambda/default_handler.py

```diff
@@ -0,0 +1,57 @@
+import os
+import boto3
+import asyncio
+from .parallel import parallel_map_on_lambda
+from botocore.exceptions import ClientError
+
+LAMBDA_WORKER_FUNCTION_NAME = "LAMBDA_WORKER_FUNCTION_NAME"
+env_lambda_worker_function_name = (
+    os.environ[LAMBDA_WORKER_FUNCTION_NAME]
+    if LAMBDA_WORKER_FUNCTION_NAME in os.environ
+    else None
+)
```

Comment:
Mappings have a convenience method for this:

```py
env_lambda_worker_function_name = os.environ.get("LAMBDA_WORKER_FUNCTION_NAME")
```

Reply:
Fixed in 41247e71a1dac446ad6f22d12b88b2e23f5ced91